src/runtime_src/core/common/api/xrt_bo.cpp

```diff
@@ -1171,7 +1173,8 @@ fill_copy_pkt(const xrt::bo& dst, const xrt::bo& src, size_t sz,
 #ifndef _WIN32
```

Comment:
At a later date, the code should be refactored to call a common XRT helper routine to do this functionality.   
As it currently is implemented, this flow is not support to run on windows.